docs/program/apis/sessions.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Sessions Management with Viam's Client SDKs"
+linkTitle: "Session Management API"
+weight: 20
+type: "docs"
+description: "How to use the Sessions Management API with Viam's Client SDKs."
```

Comment:
```suggestion
description: "How to use the session management API with Viam's Client SDKs."
```
Pick session or sessions and make all throughout here the same. Also headings -> sentence case, and lower case session management (or sessions management if you go that route)

Reply:
Sounds good to me 😆, thank you!

Reply:
done